media/templates/homepage.mustache

```diff
@@ -80,6 +80,30 @@
 
 <div class="visualclear"></div>
 
+{{#num_pages}}
+<ul class="projectlist pagination">
+{{#previous_page}}
+    <li>
+        <a href="#" aria-label="Previous" data-page="{{.}}" class="page">
+            <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+        </a>
+    </li>
+{{/previous_page}}
+    <li class="description">
+        <a href="#" aria-label="{{current_page}}/{{num_pages}}">
```

Comment:
Clicking this element doesn't do anything, and causes some weird style issues with hovering. Is there a reason that this is an `<a>` tag? I think it could just be a div.